src/Service/MarketoMaApiClientInterface.php

```diff
@@ -100,4 +100,17 @@ public function syncLead(Lead $lead, $key = 'email', $options = []);
    */
   public function deleteLead($leads, $args = array());
 
+  /**
+   * Associate a known lead with activity recorded by a cookie.
+   *
+   * @param $leadId
+   *   The id of the lead whose activity should be associated with the cookie.
+   * @param $cookie
+   *   The browser cookie value to associate with the lead.
+   * @param array $args
+   *
+   * @return mixed
+   */
+  public function associateLead($leadId, $cookie, $args = array());
```

Comment:
Args isn't a valid parameter to pass to the Marketo API, but I included it in the method signature to maintain consistency with the rest of the interface where an array of optional configuration could be passed through to influence the operation even if it was not directly passed along to Marketo.